tools/sidekick/config.json

```diff
@@ -6,7 +6,6 @@
       "title": "Library",
       "environments": [ "edit" ],
       "isPalette": true,
-      "passConfig": true,
```

Comment:
@chrischrischris when we remove this, the library shows up. But I see other repos using it without issue. Is there something obvious I'm missing that would allow the config to stay - and is it even useful right here?

Reply:
@rofe asking around, I was told you might have some input as well.